plugins/org.jkiss.dbeaver.ext.tidb/src/org/jkiss/dbeaver/ext/tidb/mysql/model/TiDBMySQLCatalog.java

```diff
@@ -0,0 +1,266 @@
+                        sql.append(" WHERE ");
+                        if (!CommonUtils.isEmpty(tableFilters.getInclude())) {
+                            sql.append("(");
+                            boolean hasCond = false;
+                            for (String incName : tableFilters.getInclude()) {
+                                if (hasCond) {
+                                    sql.append(" OR ");
+                                }
+
+                                hasCond = true;
+                                sql.append(tableNameCol).append(" LIKE ").append(
+                                        SQLUtils.quoteString(session.getDataSource(), SQLUtils.makeSQLLike(incName)));
+                            }
+                            sql.append(")");
+                        }
+                        if (!CommonUtils.isEmpty(tableFilters.getExclude())) {
+                            if (!CommonUtils.isEmpty(tableFilters.getInclude())) {
+                                sql.append(" AND ");
+                            }
+                            sql.append("(");
+                            boolean hasCond = false;
+                            for (String incName : tableFilters.getExclude()) {
+                                if (hasCond) {
+                                    sql.append(" OR ");
+                                }
+                                hasCond = true;
+                                sql.append(tableNameCol).append(" NOT LIKE ")
+                                        .append(SQLUtils.quoteString(session.getDataSource(), incName));
+                            }
+                            sql.append(")");
+                        }
+                    } else if (dataSource.supportsSequences()) {
+                        sql.append(" WHERE Table_type <> 'SEQUENCE'");
+                    }
+                }
+            }
+
+            return session.prepareStatement(sql.toString());
+        }
+
+        @Override
+        protected MySQLTableBase fetchObject(@NotNull JDBCSession session, @NotNull TiDBMySQLCatalog owner,
+                @NotNull JDBCResultSet dbResult) {
+            final String tableType = JDBCUtils.safeGetString(dbResult, MySQLConstants.COL_TABLE_TYPE);
+            if (tableType != null && tableType.contains("VIEW")) {
+                return new TiDBMySQLView(owner, dbResult);
+            } else {
+                return new MySQLTable(owner, dbResult);
+            }
+        }
+
+        @Override
+        protected JDBCStatement prepareChildrenStatement(@NotNull JDBCSession session,
+                @NotNull TiDBMySQLCatalog owner, @Nullable MySQLTableBase forTable) throws SQLException {
+            if (forTable instanceof TiDBMySQLView) {
+                JDBCPreparedStatement dbStat = session
+                        .prepareStatement("desc " + owner.getName() + "." + forTable.getName());
```

Comment:
Please use `forTable.getFullyQualifiedName(DBPEvaluationContext.DDL)` instead `owner.getName() + "." + forTable.getName()`
To avoid cases when a table or catalog is quoted.